### ISSUE TITLE
Directive shortcut to disable a single line

### ISSUE THREAD

**Issue by juliusl** [koalaman/shellcheck#2323]
#### For new checks and feature suggestions
- [ X] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

I searched for "disable shortcut" and `disable=`

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh

# shellcheck disable=
set $WANT_STRING_SPLITTING

```

#### Here's what shellcheck currently says:

SC2086 warning

#### Here's what I wanted or expected to see:

No warning

---
I use this as an example case because I like this rule so I wouldn't want to globally ignore it. So it would be great if the directive `# shellcheck disable=`  cleared any warnings on the next line. However, I understand that reduces posterity so, alternatively if there were semantics for a reason/label as an identifier, then you could do:

`# shellcheck disable='args'`

and then at the top of the file ,

`# shellcheck disable=SC2086,SC#### when='args'` 

I think this can be accomplished without having to change upstream, but in my opinion I think it would be nice syntactic sugar to have coming from the directive parser.

**Comment by koalaman** [comment]
In 0.7.2 you can do `# shellcheck disable=0-9999`, and now `disable=all` is now a shortcut for this. Thanks!

(Be aware that any directives before the first command in a script apply to the entire script and not just that command)

**Comment by juliusl** [comment]
@koalaman great thanks!
